### PR TITLE
Feature/aa02 test

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/integration/accounts/dormant_company_accounts.spec.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        // Check the Contact details and authorisation page
+        cy.accessibilityCheck();
+        contactDetailsAndAuthorisationPage.submitAuthorisation();
+        // Check the AA02 submission confiration screen
+        cy.accessibilityCheck();
+        submissionConfirmation.confirmHeadingContains('WebFiling Service Confirmation').done();
+    })
+
+    it('Check Share class selection popup screen', () => {
+        cy.visit('/select_menu?param=AA02.IssuedShareCapitalControl.IssuedShareCapitalListControl.IssuedShareCapitalItemControl_1.ShareClass');
```
Comment (rsuller) [inline cypress/integration/accounts/dormant_company_accounts.spec.js]: Slightly concerned over the length of this url but not enough to do anything about it

File: cypress/support/page_objects/DormantAccountsLandingPage.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class DormantAccountsLandingPage {
+
+    confirmHeadingContains(text) {
```
Comment (rsuller) [inline cypress/support/page_objects/DormantAccountsLandingPage.js]: One to move to Base page possibly as could be used on all pages?

Comment (danfalc) [reply cypress/support/page_objects/DormantAccountsLandingPage.js]: It is done

**Comment by danfalc** [comment]
image

I have taken out the uncaught:exception as discussed with @rsuller. An issue will be raised to cover the test failure above

**Pull request merged**